userguide/advanceduse/reverse-tethering.rst

```diff
@@ -1,34 +1,34 @@
 Reverse tethering
 =================
 
-Some users may not have an available wifi connection for their phone nor a data subscription from their mobile provider.
-This short tutorial will help you to connect your Ubuntu Touch to your computer to access internet.
+This tutorial helps you to connect your Ubuntu Touch device to your computer to access the Internet on it.
+This is useful if there is no available Wi-Fi connection or you don't have a data subscription on your Ubuntu Touch device.
 
-Prerequisite: Phone is connected to the computer with usb and developer mode enabled.
+Prerequisite: A Ubuntu Touch device with "Developer mode" on, attached via USB to the computer connected to the Internet.
 
 Steps
 -----
 
-1. On phone: ``android-gadget-service enable rndis``
+1. On the Ubuntu Touch device: ``android-gadget-service enable rndis``
```

Comment:
Does this have to be

- Device
- Computer
- Device
- Computer
?

Reply:
I guess we could do 
computer 2
phone 1
phone 3
computer 4